src/test/resources/edu/hm/hafner/coverage/parser/clover/clover.xml

```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<coverage generated="1695944532781" clover="3.2.0">
+    <project timestamp="1695944532781" name="All files">
+        <metrics statements="3057" coveredstatements="2578" conditionals="1444" coveredconditionals="1068" methods="1003" coveredmethods="736" elements="5504" coveredelements="4382" complexity="0" loc="3057" ncloc="3057" packages="68" files="170" classes="170"/>
```

Comment:
If you want to see the correct LOC you can set the LOC value on the project level (otherwise it is computed from the line values).